packages/xstate-wallet/src/chain.ts

```diff
@@ -413,22 +413,22 @@ export class ChainWatcher implements Chain {
       flatMap(async () => this.getChainInfo(channelId))
     );
 
-    // @ts-ignore -- FIXME: ethers events do not have .off for some reason
-    const timeoutEvents = fromEvent(this._adjudicator?.provider, 'block').pipe(
-      flatMap(async (blockNumber: number) => {
-        const chainInfo = await this.getChainInfo(channelId);
-        return {blockNumber, chainInfo};
-      }),
-      filter(
-        ({
-          blockNumber,
-          chainInfo: {
-            channelStorage: {finalizesAt}
-          }
-        }) => finalizesAt.gt(0) && finalizesAt.lte(blockNumber)
-      ),
-      map(({chainInfo}) => ({channelId, ...chainInfo}))
-    );
+    // FIXME: ethers events do not have .off for some reason
+    // const timeoutEvents = fromEvent(this._adjudicator?.provider, 'block').pipe(
+    //   flatMap(async (blockNumber: number) => {
+    //     const chainInfo = await this.getChainInfo(channelId);
+    //     return {blockNumber, chainInfo};
+    //   }),
+    //   filter(
+    //     ({
+    //       blockNumber,
+    //       chainInfo: {
+    //         channelStorage: {finalizesAt}
+    //       }
+    //     }) => finalizesAt.gt(0) && finalizesAt.lte(blockNumber)
+    //   ),
+    //   map(({chainInfo}) => ({channelId, ...chainInfo}))
+    // );
```

Comment:
Taken at face value, this seems a little heavy handed -- can you justify?

Is it better than just leaving the  code intact with its accompanying FIXME?

Reply:
The variable was unused and did not work anyway because you can't pass in an ethers style event emitter as an argument. Still need to work on this further.

Reply:
Opinion: If we are sure we don't use this, better delete the whole thing.